gnomad_qc/v4/resources/release.py

```diff
@@ -79,13 +79,13 @@ def combined_faf_release_ht_path(
         bucket. Default is False.
     :return: File path for desired combined genome + exome FAF release Hail Table.
     """
-    if public:
-        if file_exists(combined_faf().versions[release_version].path):
-            return combined_faf().versions[release_version].path
-        else:
-            return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/joint/gnomad.joint.v{release_version}.faf.ht"
-    else:
-        return f"gs://gnomad/release/{release_version}/ht/joint/gnomad.joint.v{release_version}.faf.ht"
+    # if public:
+    #     if file_exists(combined_faf().versions[release_version].path):
+    #         return combined_faf().versions[release_version].path
+    #     else:
+    #         return f"gs://gnomad-public-requester-pays/release/{release_version}/ht/joint/gnomad.joint.v{release_version}.faf.ht"
+    # else:
```

Comment:
This should be entirely deleted if we aren't using it but why is it being removed?

Reply:
because Julia said we will put combined_faf resources later in gnomAD_methods. Can I just quote it for later use?